From 63e2dd831da0ea5d6631e17431b68c1620e94d1b Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynhnguyen.xb@renesas.com>
Date: Fri, 24 Jul 2020 16:50:29 +0700
Subject: [PATCH 321/475] ARM: dts: r8a7743: Add SATA[01] nodes

Add definitions of SATA[01] to the R8A7743 (RZ/G1M) SoC device tree.

Signed-off-by: Quynh Nguyen <quynhnguyen.xb@renesas.com>
---
 arch/arm/boot/dts/r8a7743.dtsi | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7743.dtsi b/arch/arm/boot/dts/r8a7743.dtsi
index 7d16fa0..7edb65a 100644
--- a/arch/arm/boot/dts/r8a7743.dtsi
+++ b/arch/arm/boot/dts/r8a7743.dtsi
@@ -1142,6 +1142,24 @@
 			status = "disabled";
 		};
 
+		sata0:sata@ee300000 {
+			compatible= "renesas,sata-r8a7743";
+			reg= <0 0xee300000 0 0x2000>;
+			interrupts= <0 105 IRQ_TYPE_LEVEL_HIGH>;
+			clocks= <&mstp8_clks R8A7743_CLK_SATA0>;
+			power-domains= <&cpg_clocks>;
+			status= "disabled";
+		};
+
+		sata1:sata@ee500000 {
+			compatible= "renesas,sata-r8a7743";
+			reg= <0 0xee500000 0 0x2000>;
+			interrupts= <0 106 IRQ_TYPE_LEVEL_HIGH>;
+			clocks= <&mstp8_clks R8A7743_CLK_SATA1>;
+			power-domains= <&cpg_clocks>;
+			status= "disabled";
+		};
+
 		clocks {
 			#address-cells = <2>;
 			#size-cells = <2>;
-- 
2.7.4

